clients/go/pkg/justification/processor.go

```diff
@@ -90,6 +92,23 @@ func (p *Processor) Process(ctx context.Context, logReq *api.AuditLogRequest) er
 	}
 
 	logReq.Payload.Metadata.Fields[LogMetadataKey] = structpb.NewStructValue(&tokStruct)
-	// TODO(#266): Also populate RequestAttribute.Reason.
+
+	justs, ok := (*tok).Get("justs")
```

Comment:
I'm not sure I understand why we need to dereference this? Same with the call to json.Marshal above.

Reply:
The return of function `ValidateJWT` is a pointer of token
 pointer https://github.com/abcxyz/lumberjack/blob/29ad85da8bbc00812ba4023576d9022c3f53b0a0/clients/go/pkg/justification/processor.go#L42? Do you mean we should change the interface of `ValidateJWT` here?

Reply:
I think the question is why we need to dereference the pointer. Why can't we use it directly. TBH I didn't even pay attention to that :P

Reply:
`Get` is a method of interface `Token` instead of interface of Token Pointer. To call `Get`, I need to deference to the pointer.

Reply:
You can call the function regardless of the receiver: https://go.dev/play/p/dyDG1YNBU0U

We should also always aim to define functions on the pointer receivers.

If `jwt.Token` is an interface (looks like it is), we should update our interface to return the interface, not the pointer to the interface.

Reply:
I think I see what's going on: https://github.com/abcxyz/jvs/pull/117

Reply:
And https://github.com/abcxyz/jvs/issues/116